src/nvtx3/nvtxDetail/nvtxImpl.h

```diff
@@ -0,0 +1,438 @@
+/*
+* Copyright 2009-2022  NVIDIA Corporation.  All rights reserved.
+*
+* Licensed under the Apache License v2.0 with LLVM Exceptions.
+* See https://llvm.org/LICENSE.txt for license information.
+* SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+*/
+
+#ifndef NVTX_IMPL_GUARD
+#error Never include this file directly -- it is automatically included by nvToolsExt.h (except when NVTX_NO_IMPL is defined).
+#endif
+
+/* ---- Include required platform headers ---- */
+
+#if defined(_WIN32) 
+
+#include <Windows.h>
```

Comment:
I know this isn't your code, but this should be
```suggestion
#include <windows.h>
```
:smiling_face_with_tear:

Reply:
https://github.com/NVIDIA/NVTX/pull/63